app/controllers/recipes_controller.rb

```diff
@@ -0,0 +1,6 @@
+class RecipesController < ApplicationController
+  def show
+    @recipe = Recipe.find(params[:id])
+    @recipe_ingredients = RecipeIngredient.find_ingredients(@recipe.id)
```

Comment:
I believe this works, but if you're trying to find all the ingredients that belong to this recipe, why not use its association? `@recipe.ingredients` does the same thing for way cheaper.

Reply:
🤯